packages/storybook-vue/src/stories/Link/Link.stories.mdx

```diff
@@ -0,0 +1,187 @@
+
+<Canvas>
+  <Story
+    name="External"
+    args={{ ...componentArgs, default: "Extenal component", external: true, target: "_blank", rel: "noopener" }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Link :external="true" target="_blank" rel="noopener">External</Link>`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Eamil
+
+<Canvas>
+  <Story
+    name="Email"
+    args={{ ...componentArgs, default: "Email", href: "mailto:info@example.com" }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Link href="mailto:info@example.com">Email</Link>`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Telephone
+
+<Canvas>
+  <Story
+    name="Telephone"
+    args={{ ...componentArgs, default: "+31 555 385 385", href: "tel:+31555385385" }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Link href="tel:+31555385385">+31 555 385 385</Link>`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Alternate Language
+
+<Canvas>
+  <Story
+    name="Alternate Language"
+    args={{
+      ...componentArgs,
+      default: "EN",
+      href: "https://example.com/en",
+      title: "This page in English",
+      hreflang: "en",
+      lang: "en",
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Link 
+          title="This page in English" 
+          hreflang="en" 
+          lang="en" 
+          href="https://example.com/en"
+          >
+          EN
+          </Link>`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Current language
+
+<Canvas>
+  <Story
+    name="Current language"
+    args={{
+      ...componentArgs,
+      default: "EN",
+      href: "https://example.com/en",
+      "aria-current": "page",
+      title: "This page in English",
+      hreflang: "en",
+      lang: "en",
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `<Link 
+          href="https://example.com/en" 
+          aria-current="page" 
+          title="This page in English" 
+          hreflang="en" 
+          lang="en"
+          >
+          EN
+          </Link>`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Eamil
+
+<Canvas>
+  <Story
+    name="Arabic Language"
+    args={{
+      ...componentArgs,
+      default: "تواصل معنا من خلال البريد الإلكتروني",
+      href: "mailto:info@example.com",
+      dir: "rtl",
+      lang: "ar",
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: `
+          <div dir="rtl" lang="ar"> 
+          <Link
+          dir="rtl"
+          lang="ar"
+          href="mailto:info@example.com"
+          >
+          تواصل معنا من خلال البريد الإلكتروني
+          </Link>
+          </div>
+          `,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
```

Comment:
I think we can remove the Arabic example from the Vue docs, since it was mainly included to tweak the CSS `text-underline-offset`, there is no special API or class name for this.